### PR TITLE
Templates for stsb_multi_mt_en

### PR DESCRIPTION
**Pull request by NohTow**
I tried to write templates for stsb_multi_mt (the english subset).
There is some that ask for the numerical score and others that are in the form of yes/no using a threshold (see my question on slack).

### CODE REVIEW

File: templates/stsb_multi_mt/en/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  9cab340c-32ce-465d-be89-049e4a63af11: !Template
+    id: 9cab340c-32ce-465d-be89-049e4a63af11
+    jinja: On a scale from {{"1.0"}} to {{"5.0"}}, how similar "{{sentence1}}" and
+      "{{sentence2}}" are ? ||| {{((5*similarity_score)|round)/5}}
+    name: Similarity_scoring_1
+    reference: ''
+  e0551bee-61f0-4c1e-9c3f-18c8b54439f8: !Template
+    id: e0551bee-61f0-4c1e-9c3f-18c8b54439f8
+    jinja: "Rate the similarity of these two sentences: ({{\"1.0\"}} being the lowest\
+      \ and {[\"5.0\"}} the highest)\n\"{{sentence1}}\" and \"{{sentence2}}\" \n|||\n\
```
Comment (stephenbach) [inline templates/stsb_multi_mt/en/templates.yaml:31]: Should be `{{"5.0"}}`

File: templates/stsb_multi_mt/en/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  775af665-d8a5-46b2-bfcf-2a21abc7e99c: !Template
+    id: 775af665-d8a5-46b2-bfcf-2a21abc7e99c
+    jinja: Do "{{sentence1}}" and "{{sentence2}}" seem similar to you ? ||| {{["no","yes"][0
+      if similarity_score < 2.5 else 1]}}'
+    name: Similarity_scoring_5
+    reference: ''
+  9cab340c-32ce-465d-be89-049e4a63af11: !Template
+    id: 9cab340c-32ce-465d-be89-049e4a63af11
+    jinja: On a scale from {{"1.0"}} to {{"5.0"}}, how similar "{{sentence1}}" and
+      "{{sentence2}}" are ? ||| {{((5*similarity_score)|round)/5}}
```
Comment (stephenbach) [inline templates/stsb_multi_mt/en/templates.yaml:25]: Maybe more natural to say "how similar are X and Y?"

File: templates/stsb_multi_mt/en/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  6df357b5-f8ea-49d2-b304-3541acb5271a: !Template
+    id: 6df357b5-f8ea-49d2-b304-3541acb5271a
+    jinja: Do you think "{{sentence1}}" and "{{sentence2}}" express the same thing?
+      ||| {{["no","yes"][0 if similarity_score < 2.5 else 1]}}'
+    name: Similarity_scoring_4
+    reference: sst2
+  775af665-d8a5-46b2-bfcf-2a21abc7e99c: !Template
+    id: 775af665-d8a5-46b2-bfcf-2a21abc7e99c
+    jinja: Do "{{sentence1}}" and "{{sentence2}}" seem similar to you ? ||| {{["no","yes"][0
+      if similarity_score < 2.5 else 1]}}'
```
Comment (stephenbach) [inline templates/stsb_multi_mt/en/templates.yaml:19]: Extra `'` on the end

File: templates/stsb_multi_mt/en/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  6c0bdf61-9baa-415a-bf03-fdb8789d1740: !Template
+    id: 6c0bdf61-9baa-415a-bf03-fdb8789d1740
+    jinja: How similar are "{{sentence1}}" and "{{sentence2}}"? Give a score between
+      {{"0.0"}} and {{"5.0"}}. ||| {{((5*similarity_score)|round)/5}}
+    name: Similarity_scoring_3
+    reference: ''
+  6df357b5-f8ea-49d2-b304-3541acb5271a: !Template
+    id: 6df357b5-f8ea-49d2-b304-3541acb5271a
+    jinja: Do you think "{{sentence1}}" and "{{sentence2}}" express the same thing?
+      ||| {{["no","yes"][0 if similarity_score < 2.5 else 1]}}'
```
Comment (stephenbach) [inline templates/stsb_multi_mt/en/templates.yaml:13]: Extra `'` on the end

**Review by stephenbach — CHANGES_REQUESTED** [review]
Great, thanks! Just a few minor clean ups and suggestions.

**Comment by NohTow** [comment]
Fixed these issues (sorry for the typos, I don't know why I didn't see them...) and checked the main task box.

**Comment by stephenbach** [comment]
Perfect, thanks!

**Pull request merged**